chromadb/proto/logservice_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: chromadb/proto/logservice.proto
-# Protobuf Python Version: 5.26.1
+# Protobuf Python Version: 4.25.1
```

Comment:
might want to bump your protoc version and re-gen

Reply:
That's the grpc version when I run `pip install` in a fresh python venv given `requirements.txt` and `dev_requirements.txt`, and it cannot be updated due to compatibility issues. I'm wondering if it is safe to use a new version here.

Reply:
weird, ok
maybe whoever generated this last had an incorrect version then